Makefile

```diff
@@ -64,38 +43,34 @@ run:
 	KUBERNETES_CONFIG=$(KUBECONFIG) \
 	WORKING_DIR=$(TARGET_DIR)/ocp-clo \
 	LOGGING_SHARE_DIR=$(CURDIR)/files \
-	go run ${MAIN_PKG}
+	go run cmd/manager/main.go
 
 clean:
-	@rm -rf $(TARGET_DIR) && \
-	go clean -cache -testcache  $(TEST_PKGS) $(PKGS)
+	@rm bin/*
+	go clean -cache -testcache ./...
 
-image: imagebuilder
+image: bin/imagebuilder build
 	@if [ $${USE_IMAGE_STREAM:-false} = false ] && [ $${SKIP_BUILD:-false} = false ] ; \
 	then hack/build-image.sh $(IMAGE_TAG) $(IMAGE_BUILDER) $(IMAGE_BUILDER_OPTS) ; \
 	fi
 
-lint:
+lint: fmt
 	golangci-lint run -c golangci.yaml
```

Comment:
@alanconway `make build` will fail if this `golangci-lint` is missing. I guess we should add this as a prerequisite to README.md, or add the way to download it automatically if it is missing.

Reply:
+1, fixing.